src/resources/namespaced/organization-organizations-venues.ts

```diff
@@ -21,14 +21,14 @@ class OrganizationOrganizationsVenues extends ResourceClass {
 	/**
 	 * Add an organization venue link
 	 * @params {id:uuid, venue_id:uuid}
-	 * @required {id:uuid, venue_id:uuid}
-	 * @url /organizations/{id}/organizations_venues
+	 * @required {id:uuid, venue_id:uuid, organization_id: uuid}
+	 * @url /organizations/{id}/organization_venues
 	 */
 	create(): OrganizationInterface {
 		return createRequestMethod({
 			method: "POST",
-			path: "/{id}/organizations_venues",
-			required: ["venue_id"],
+			path: "/{id}/organization_venues",
```

Comment:
If you change the `{id}` to `{organization_id}` the system knows to replace it and keep it in the data because it is now required. That way you don't need to pass in an object with `{id, venue_id, organization_id}` which could lead to confusion about which `id` we're talking about
```suggestion
			path: "/{organization_id}/organization_venues",
```